ArjunaJTA/object_store/pom.xml

```diff
@@ -35,6 +35,11 @@
   <packaging>jar</packaging>
   <name>Configuring the Object Store</name>
 
+  <properties>
+    <version.com.h2database>1.4.195</version.com.h2database>
```

Comment:
What about using https://github.com/jbosstm/quickstart/blob/main/pom.xml#L196 ?

Reply:
Quickstarts are supposed to be self-contained.
But if the general feeling (@jmfinelli ) is to centralise versions then I'll do what you ask.

Reply:
But a related point is that we need boms for the main narayana repo

Reply:
@marcosgopen So I am sticking with version `1.4.195` since I want to be consistent with what we test against in the main narayana repo (we use H2 for the transaction log storage). Will you reconsider the comments that you have against this PR?

Reply:
Since narayana is using version 1.4.195 I reconsider my previous comment and I approve this PR. Thanks again for this PR @mmusgrov 
Though I created a Jira for upgrading h2 dependency (https://issues.redhat.com/browse/JBTM-3736)